src/quality.spec.js

```diff
@@ -1,7 +1,7 @@
 const nodeHtmlToImage = require('./index.js')
 const puppeteer = require('puppeteer')
 
-describe('quality', () => {
+describe.skip('quality', () => {
```

Comment:
Why are we skipping this one, though?

Reply:
I need to fix it before merging this pull request. It doesn't work with the puppeteer-cluster implementation 👌

Reply:
Great! Everything looks fine on my end, so go ahead and merge whenever you're ready 🚀